docs/CSBR.md

```diff
@@ -212,338 +586,939 @@ copyright: |
 
 ### 4.9.16 Limits on suspension period
 
-## 4.10  Certificate status services
+## 4.10 Certificate status services
 
 ### 4.10.1 Operational characteristics
 
 ### 4.10.2 Service availability
 
 ### 4.10.3 Optional features
 
-## 4.11  End of subscription
+## 4.11 End of subscription
 
-## 4.12  Key escrow and recovery
+## 4.12 Key escrow and recovery
 
 ### 4.12.1 Key escrow and recovery policy and practices
 
 ### 4.12.2 Session key encapsulation and recovery policy and practices
 
-# 5. FACILITY, MANAGEMENT, AND OPERATIONAL CONTROLS (11)
+# 5. FACILITY, MANAGEMENT, AND OPERATIONAL CONTROLS
+
+## 5.1 Physical controls
+
+### 5.1.1 Site location and construction
+
+### 5.1.2 Physical access
+
+### 5.1.3 Power and air conditioning
+
+### 5.1.4 Water exposures
+
+### 5.1.5 Fire prevention and protection
+
+### 5.1.6 Media storage
+
+### 5.1.7 Waste disposal
+
+### 5.1.8 Off-site backup
+
+## 5.2 Procedural controls
+
+### 5.2.1 Trusted roles
+
+### 5.2.2 Number of persons required per task
+
+### 5.2.3 Identification and authentication for each role
+
+### 5.2.4 Roles requiring separation of duties
+
+## 5.3 Personnel controls
+
+As specified in EV Guidelines Section 14.1.

```

Comment:
As written I interpret this as being required for both Non EV and EV CSC, is that the intent?

Reply:
Yes, the intent is to encompass both OVCS and EVCS, as that is the current requirement in CSBR 14.1 as of 2021-06-01. Do you think there's value in explicitly stating that this requirement is applicable to both OVCS and EVCS?

Reply:
Unless I'm misreading your suggestion regarding section 14.1.3, I think that 4.2.1 will be sufficiently clear that at least 2 Validation Specialists must validate and review the certificate request: "Prior to issuing Code Signing Certificates, the CA SHALL perform "due diligence" verification as specified in EV Guidelines 11.13.". Given this, I'm not sure we need to repeat that requirement here.

> The CA SHALL require all Validation Specialists to pass an examination provided by the CA on the information verification requirements outlined in these Requirements.

I like this language; I'll add in the next commit.

Reply:
My reasoning for the call out is simply clarity. A reader unfamiliar with requirements may confuse references to EVG to be applicable to EV Code Signing only. I don't feel too strongly about this so I'm good either way.